umi_tools/extract.py

```diff
@@ -425,6 +425,11 @@ def main(argv=None):
             options.stdout.write(str(new_read) + "\n")
 
     else:
+        #variable for progress monitor
+        progCount = 0
+        displayMax = 100000
+        print("Starting barcode extraction")
```

Comment:
Yep, sounds good.
I was just trying to point it out as an example since there can be other such places where progress monitoring make sense, specially with 10x data where number of cells are easily above 1000 making library size even bigger.